### PR TITLE
Feature/ios remaining player api functions

### PR DESCRIPTION
**Pull request by fotiDim**
☝️What the name says. This is the last PR to be merged.
I added 2 more methods to the dart API which I should also be implemented for Android. On iOS there are no `toggle` functions. I updated the readme accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,21 +106,22 @@ Have a look [in the example](example/lib/main.dart) for detailed insights on how
 
 | Function  | Description | Android | iOS | Web |
 |---|---|---|---|---|
-|  getCrossfadeState | Gets the current crossfade state | :heavy_check_mark:  |  :construction_worker: | :x: |
```
Comment (fotiDim) [inline README.md:109]: This was a duplicate

File: ios/Classes/SpotifySdkConstants.swift [hunk 2]
```diff
@@ -19,13 +19,13 @@ public class SpotfySdkConstants
     public static let methodPlay = "play"
     public static let methodPause = "pause"
     public static let methodResume = "resume"
-    public static let methodToggleRepeat = "toggleRepeat"
-    public static let methodToggleShuffle = "toggleShuffle"
     public static let methodSkipNext = "skipNext"
     public static let methodSkipPrevious = "skipPrevious"
     public static let methodSeekTo = "seekTo"
-    public static let methodSeekToRelativePosition = "seekToRelativePosition"
```
Comment (brim-borium) [inline ios/Classes/SpotifySdkConstants.swift:27]: What happened to seekToRelativePosition?

File: lib/spotify_sdk.dart [hunk 3]
```diff
@@ -496,6 +496,39 @@ class SpotifySdk {
     }
   }
 
+  /// Sets the shuffle mode
+  ///
+  /// Set [shuffle] to true or false.
+  /// Throws a [PlatformException] if adding failed
+  /// Throws a [MissingPluginException] if the method is not implemented on
+  /// the native platforms.
+  static Future setShuffle({@required bool shuffle}) async {
```
Comment (brim-borium) [inline lib/spotify_sdk.dart:505]: Is this iOS only ?

Comment (brim-borium) [reply lib/spotify_sdk.dart:505]: Ah nevermind I am an idiot :D

File: README.md [hunk 4]
```diff
@@ -106,21 +106,22 @@ Have a look [in the example](example/lib/main.dart) for detailed insights on how
...
+|  seekToRelativePosition | Adds to the current position of the track the given milliseconds |:x: |  :heavy_check_mark: | :construction_worker: |
 |  subscribeToPlayerContext | Subscribes to the current player context |:heavy_check_mark:|:heavy_check_mark: | :heavy_check_mark: |
 |  subscribeToPlayerState| Subscribes to the current player state |:heavy_check_mark:  | :heavy_check_mark: | :heavy_check_mark: |
-|  getCrossfadeState | Gets the current crossfade state |:heavy_check_mark:  |  :construction_worker: | :x: |
-|  toggleShuffle | Cycles through the shuffle modes |:heavy_check_mark: |  :construction_worker: | :construction_worker: |
-|  toggleRepeat | Cycles through the repeat modes | :heavy_check_mark: |  :construction_worker: | :construction_worker: |
+|  getCrossfadeState | Gets the current crossfade state |:heavy_check_mark:  |  :heavy_check_mark: | :x: |
+|  toggleShuffle | Cycles through the shuffle modes |:heavy_check_mark: |  :x: | :construction_worker: |
+|  setShuffle | Set the shuffle mode |:construction_worker::construction_worker: |  :construction_worker: | :construction_worker: |
```
Comment (brim-borium) [inline README.md:122]:
```suggestion
|  setShuffle | Set the shuffle mode | :construction_worker: |  :construction_worker: | :construction_worker: |
```

Comment (brim-borium) [reply README.md:122]: General question: SetShuffle should work for iOS right? So we probably need a checkmark here. Same for setRepeatMode

File: README.md [hunk 5]
```diff
@@ -106,21 +106,22 @@ Have a look [in the example](example/lib/main.dart) for detailed insights on how
...
-|  queue | Queues given spotifyUri |:heavy_check_mark: | :construction_worker:  | :heavy_check_mark: |
+|  queue | Queues given spotifyUri |:heavy_check_mark: | :heavy_check_mark:  | :heavy_check_mark: |
 |  resume | Resumes the current track |:heavy_check_mark: |  :heavy_check_mark: | :heavy_check_mark: |
 |  skipNext | Skips to next track | :heavy_check_mark: | :heavy_check_mark:  | :heavy_check_mark: |
 |  skipPrevious | Skips to previous track |:heavy_check_mark:  |  :heavy_check_mark: | :heavy_check_mark: |
-|  seekTo | Seeks the current track to the given position in milliseconds | :heavy_check_mark: |:construction_worker: | :construction_worker: |
-|  seekToRelativePosition | Adds to the current position of the track the given milliseconds |:heavy_check_mark: |  :construction_worker: | :construction_worker: |
+|  seekTo | Seeks the current track to the given position in milliseconds | :heavy_check_mark: |:heavy_check_mark: | :construction_worker: |
+|  seekToRelativePosition | Adds to the current position of the track the given milliseconds |:x: |  :heavy_check_mark: | :construction_worker: |
```
Comment (brim-borium) [inline README.md:117]: Does Android not work anymore for seekToRelative?

**Review by brim-borium — CHANGES_REQUESTED** [review]

File: lib/spotify_sdk.dart (hunk 3 above)
Comment (fotiDim) [reply lib/spotify_sdk.dart:505]: Android as well. We should implement it at some point.

File: README.md [hunk 6]
```diff
@@ -106,21 +106,22 @@ Have a look [in the example](example/lib/main.dart) for detailed insights on how
...
 |  subscribeToPlayerState| Subscribes to the current player state |:heavy_check_mark:  | :heavy_check_mark: | :heavy_check_mark: |
-|  getCrossfadeState | Gets the current crossfade state |:heavy_check_mark:  |  :construction_worker: | :x: |
-|  toggleShuffle | Cycles through the shuffle modes |:heavy_check_mark: |  :construction_worker: | :construction_worker: |
-|  toggleRepeat | Cycles through the repeat modes | :heavy_check_mark: |  :construction_worker: | :construction_worker: |
+|  getCrossfadeState | Gets the current crossfade state |:heavy_check_mark:  |  :heavy_check_mark: | :x: |
+|  toggleShuffle | Cycles through the shuffle modes |:heavy_check_mark: |  :x: | :construction_worker: |
+|  setShuffle | Set the shuffle mode |:construction_worker::construction_worker: |  :construction_worker: | :construction_worker: |
+|  toggleRepeat | Cycles through the repeat modes | :heavy_check_mark: |  :heavy_check_mark: | :construction_worker: |
+|  setRepeatMode | Set the repeat mode | :construction_worker: |  :construction_worker: | :construction_worker: |
```
Comment (fotiDim) [inline README.md:124]:
```suggestion
|  setRepeatMode | Set the repeat mode | :construction_worker: |  :heavy_check_mark: | :construction_worker: |
```

File: README.md (hunk 4 above)
Comment (fotiDim) [reply README.md]: Good catch. Did it for both.

File: README.md (hunk 5 above)
Comment (fotiDim) [reply README.md:117]: What the hell did I do? I meant to put the `:x:` on iOS since this function does not exist.

Comment (fotiDim) [inline README.md:117]:
```suggestion
|  seekToRelativePosition | Adds to the current position of the track the given milliseconds |:heavy_check_mark: |  :x: | :construction_worker: |
```

File: ios/Classes/SpotifySdkConstants.swift (hunk 2 above)
Comment (fotiDim) [reply ios/Classes/SpotifySdkConstants.swift:27]: Not existing on iOS

File: lib/spotify_sdk.dart (hunk 3 above)
Comment (brim-borium) [reply lib/spotify_sdk.dart:505]: #41 Added an issue for that

**Review by brim-borium — APPROVED** [review]

**Pull request merged**